fv3core/pace/fv3core/stencils/fv_dynamics.py

```diff
@@ -531,7 +533,7 @@ def post_remap(
         self,
         state: DycoreState,
         is_root_rank: bool,
-        da_min: FloatFieldIJ,
+        da_min: float,
```

Comment:
Should this be float, or is it `pace.dsl.typing.Float`?

Reply:
It's an actual float, per DampingCoefficient.da_min definition